### PR TITLE
CA-218 - Makes service access security group visible in the render fille

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
* Allows `emr_service_access_sg_id` to make it visible in the render file.

### CODE REVIEW

**Review by souza-dan — APPROVED** [review]
LGTM

File: examples/ephemeral-spark/tamr-config.tf [hunk 1]
```diff
@@ -64,7 +64,7 @@ module "tamr-config" {
   emr_managed_core_sg_id      = module.ephemeral-spark-sgs.emr_managed_sg_id
   # emr_managed_core_sg_id   = "" # you may leave this blank and AWS creates one automatically
   emr_additional_core_sg_id = join(",", module.aws-emr-sg-master.security_group_ids)
```
Comment (souza-dan) [inline examples/ephemeral-spark/tamr-config.tf:66]: This is the security group for the EMR HBase cluster, which won't include the spark ports. Isn't that an issue?

Comment (fvazquez-caylent) [reply examples/ephemeral-spark/tamr-config.tf:66]: All our tests were successful ran. The Ephemeral Spark cluster spun up, ran the tasks, and destroyed. The configured service access security group allows ingress traffic from the EMR and Spark security groups on port 9443 and the outbound to the same EMR/Spark security groups to port 8443.

**Pull request merged**